aspnetcore/mvc/models/file-uploads/samples/5x/largeFiles/LargeFilesSample/Program.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace LargeFilesSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.ConfigureKestrel((context, options) =>
+                    {
+                        // ��������Ĵ�С���ƣ�ʹ�ÿ����ϴ����ļ�
+                        options.Limits.MaxRequestBodySize = long.MaxValue;
```

Comment:
Setting it to long.MaxValue is really problematic. But can we simply document configuring the server limits in the docs instead?